amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseManagementConfig.java

```diff
@@ -108,6 +112,29 @@
      */
     private boolean cleanupLeasesUponShardCompletion = true;
 
+    /**
+     * Interval to run lease cleanup thread in {@link LeaseCleanupManager}.
+     *
+     * <p>Default value: 1 hour.</p>
+     */
+    private final long leaseCleanupIntervalMillis = DEFAULT_LEASE_CLEANUP_INTERVAL_MILLIS;
```

Comment:
As we discussed on chime this can be 1 minute, which will be used by the cleanup thread scheduling.

Reply:
Updated